pcdsdevices/rs_powersupply.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+
+logger = logging.getLogger(__name__)
+
+
+class RSchannel(Device):
+    """
+    Class to define PVs of all channels of the RSPowerSupply NGP800.
+    prefix: str
+        The base PV of the relevant RohdeSchwarzPowersupply.
+    name: str, keyword-only
+        A name to refer to the relevant RohdeSchwarzPowersupply
```

Comment:
formatting and channel vs powersupply
```suggestion
    prefix : str
        The base PV of the relevant Rohde-Schwarz channel.
    name : str, keyword-only
        A name to refer to the relevant Rohde-Schwarz channel.
```